src/main/java/net/coru/kloadgen/randomtool/generator/StatelessGeneratorTool.java

```diff
@@ -72,6 +79,8 @@ public final Object generateArray(final String fieldName, final String fieldType
       result = randomArray.generateArray(fieldType, valueLength, parameterList, arraySize, Collections.emptyMap());
       if ("seq".equals(fieldType)) {
```

Comment:
should this if/else if be replicated in line 67 for generateMap method?

Reply:
We don't have implemented those structures, since we don't find a use case where we would need them.